CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 * [FEATURE] Server: added experimental [PROXY protocol support](https://www.haproxy.org/download/2.3/doc/proxy-protocol.txt). The PROXY protocol support can be enabled via `-server.proxy-protocol-enabled=true`. When enabled, the support is added both to HTTP and gRPC listening ports. #7698
 * [FEATURE] mimirtool: Add `runtime-config verify` sub-command, for verifying Mimir runtime config files. #8123
 * [FEATURE] Query-frontend, querier: new experimental `/cardinality/active_native_histogram_metrics` API to get active native histogram metric names with statistics about active native histogram buckets. #7982 #7986 #8008
+* [FEATURE] Ingester: add experimental support for the server-side circuit breakers when writing to ingesters via `-ingester.circuit-breaker.enabled`, `-ingester.circuit-breaker.failure-threshold`, or `-ingester.circuit-breaker.cooldown-period` or their corresponding YAML. Added metrics `cortex_ingester_circuit_breaker_results_total` and `cortex_ingester_circuit_breaker_transitions_total`. #8180
```

Comment:
```suggestion
* [FEATURE] Ingester: add experimental support for the server-side circuit breakers when writing to ingesters. This can be enabled using `-ingester.circuit-breaker.enabled` option. Further `-ingester.circuit-breaker.*` options for configuring circuit-breaker are available. Added metrics `cortex_ingester_circuit_breaker_results_total`,  `cortex_ingester_circuit_breaker_transitions_total` and `cortex_ingester_circuit_breaker_current_state`. #8180
```

Reply:
Done